integrations/zendesk/src/actions/make-api-request.ts

```diff
@@ -0,0 +1,42 @@
+import axios, { AxiosRequestConfig } from 'axios'
+import { getZendeskClient } from '../client'
+import * as bp from '.botpress'
+
+export const makeApiRequest: bp.IntegrationProps['actions']['makeApiRequest'] = async ({
+  ctx,
+  input,
+}): Promise<bp.actions.makeApiRequest.output.Output> => {
+  const { method, path, headers, params, requestBody } = input
+  const client = getZendeskClient(ctx.configuration)
+
+  try {
+    const requestConfig: AxiosRequestConfig = {
+      method,
+      url: `/api/v2/${path}`,
```

Comment:
**chore:** your documentation says `URL Path (https://<subdomain>.zendesk.com/api/PATH)`; update one of both